components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/SAMLAssertionClaimsCallbackTest.java

```diff
@@ -23,16 +23,23 @@
 import org.opensaml.saml2.core.Assertion;
```

Comment:
Add correct license.

Reply:
What do you mean by this? aren't we use "org.opensaml.saml2.core" ?

Reply:
Sorry I missed it.